packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -301,13 +301,35 @@ export const UploadControls = (): JSX.Element => {
     [direction, selection]
   );
 
+  const disabled = tasks.some((task) => task.status === 'PENDING');
+
   return (
     <>
       {fileSelect}
       <Title />
+      <ButtonElement
```

Comment:
Can we fix the order of these elements? These should come after the Primary/Exit buttons in the DOM order.

Also, if you put the Add files button last it should fix the misalignment.

Reply:
ah thank you Heather! Just pushed up the changes. It looks like this now:

<img width="618" alt="image" src="https://github.com/user-attachments/assets/ca7d3d80-be6c-4be5-a7a0-9dfcd4baa3b3">